paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUITest.kt

```diff
@@ -293,4 +296,24 @@ internal class PaymentMethodVerticalLayoutUITest {
     private data class Scenario(
         val viewActionRecorder: ViewActionRecorder<PaymentMethodVerticalLayoutInteractor.ViewAction>,
     )
+
+    internal companion object {
```

Comment:
Can these be `private` rather than `internal`?

Reply:
No because it's required in the test class constructor. I refactored to use an array so the companion object can be private